include/faabric/util/config.h

```diff
@@ -26,6 +26,7 @@ class SystemConfig
     std::string captureStdout;
     std::string stateMode;
     std::string wasmVm;
+    std::string deltaSnapshotEncoding;
```

Comment:
Could you add a very simple test for setting and unsetting this value in `tests/test/util/test_config.cpp`?

Reply:
Yes that's the only thing I'd add, but I'll be doing some work on this so happy to add it myself. Let's approve this and I'll add that part.